### PR TITLE
Setup repository

### PR DESCRIPTION
**Pull request by kensayo**
- Setup Repository
- Add `books.js` and `categories.js` components
- Add basic navigation

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @kensayo, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      <h2>ADD NEW BOOK</h2>
+      <form>
+        <input type="text" placeholder="Book title" />
+        <select type="select">
+          <option value="test">Lorem 1</option>
+          <option value="test">Lorem 2</option>
+          <option value="test">Lorem 3</option>
+        </select>
+        <input type="submit" value="Add Book" />
+      </form>
```
Comment (mke2111) [inline src/redux/books/books.js]: - [ ] Make this another separate component in a different file. You will only need to import it to the main page.
- [ ] This component should render a form with inputs for both the title and author of the book. You're currently having only the title input. Add the author input to your form.
- [ ] Add this component to the components directory and not the redux directory

File: src/redux/books/books.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        <div>Animated Circle</div>
+        <div>XX%</div>
+        <h4>Completed</h4>
+      </div>
+      <div className="px-3">
+        <h3>CURRENT CHAPTER</h3>
+        <p>Chapter 17</p>
+        <button type="button" className="btn btn-primary">Update Progress</button>
+      </div>
+    </div>
```
Comment (mke2111) [inline src/redux/books/books.js]: - [ ] You should have a BookList component that renders a list of books using the Book component, so this means you need to create both the BookList component and the Book component. Separate this file to enable these changes. Remember that the Book component should take the book data as a prop and render the title and the author.
- [ ] Add these components to the components directory and not the redux directory

File: src/redux/categories/categories.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Categories = () => (
+  <div className="w-100">
+    <h1 className="text-align-self">Site Under Construction</h1>
+  </div>
+);
+
+export default Categories;
```
Comment (mke2111) [inline src/redux/categories/categories.js]: - [ ] Move this component to the components directory. Remember that all components should be placed in the components directory.

Comment (kensayo) [reply src/redux/categories/categories.js]: But according to this, `categories.js` should be inside the Categories folder. In that case, should also move the `Books` components? Oh, or those folders are only for Redux files?

Comment (mke2111) [reply src/redux/categories/categories.js]: Those files in the redux folder will contain the redux logic only i.e actions and action types.

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @kensayo, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**